### ISSUE TITLE
Running Locally

### ISSUE THREAD

**Issue by MarkForesta** [aidenwallis/nowplaying#1]
Congrats on the success of this product where your servers are getting timed out. So much success is tough to deal with I am sure. I was hoping to alleviate some of the pain on your server I could run the javascript locally and then simply load a browser source from local host. Is this something you have thought about and you feel you should support?

**Comment by aidenwallis** [comment]
Hey! I appreciate the support, but that’s sadly not the problem, it’s to do with our client ID being ratelimited instead

**Comment by MrLucado** [comment]
Any chance you will release the source code for local hosting?

**Comment by ghost** [comment]
Would be awesome!

**Comment by MiahFuta** [comment]
Yes, please release the [spotify.js](https://github.com/aidenwallis/nowplaying/blob/master/spotify.js) source code @aidenwallis

**Comment by katawajojo** [comment]
With a few changes to the code, you can run this in conjunction with Snip to make a locally running version. A benefit to this is that you can decrease the polling time for more instantaneous updates. However, Snip doesn't always update the playing track speedily, or can just outright not update the currently playing track. I'm assuming this is still due to the Spotify API.

![2020-09-21_06-32-01](https://user-images.githubusercontent.com/13071927/93773037-6ab35e80-fbd4-11ea-9370-a361a4bbc4d9.gif)
